classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -14,10 +14,11 @@ class WC_REST_Connect_Address_Normalization_Controller extends WC_REST_Connect_B
 	public function post( $request ) {
 		$data    = $request->get_json_params();
 		$address = $data['address'];
+		$name    = $address['name'];
 		$company = $address['company'];
```

Comment:
I believe these two variables can be removed because they aren't used anywhere.

Reply:
nice catch! Thanks!